core/src/ldml/C7043_ldml.md

```diff
@@ -419,8 +425,9 @@ Entries are sorted in a binary codepoint sort on the `to` field.
 | 4 |  32  | size        | int: Length of section                   |
 | 8 |  32  | keyCount    | int: Number of keys                      |
 |12 |  32  | flicksCount | int: Number of flick lists               |
-|12 |  32  | flickCount  | int: Number of flick elements            |
-|16 | var  | keys        | keys sub-table                           |
+|16 |  32  | flickCount  | int: Number of flick elements            |
+|20 |  96  | reserved    | padding                                  |
```

Comment:
There's also ease of visually reading the data file. But that is diminished these days. 

=> https://github.com/keymanapp/keyman/issues/7987 I'd like to make this a separate issue because there's varied impact.